<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FluxPlayerClass</class>
 <widget class="QMainWindow" name="FluxPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fluxplayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="FXPlayerControlWidget" name="playerControlWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>110</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="FXTableView" name="playlistView"/>
      </item>
      <item>
       <widget class="FXVerticalAudioMeterWidget" name="levelMeter" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Playlist"/>
    <addaction name="actionOpen_playlist"/>
    <addaction name="actionSave_playlist"/>
    <addaction name="actionSave_Playlist_As"/>
    <addaction name="separator"/>
    <addaction name="actionAudio_Settings"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Playlist">
    <property name="title">
     <string>&amp;Playlist</string>
    </property>
    <addaction name="actionAddMedia"/>
    <addaction name="actionAddMediaFolder"/>
    <addaction name="actionAdd_Media_URL"/>
    <addaction name="separator"/>
    <addaction name="actionRepeat"/>
    <addaction name="actionAuto_Populate"/>
    <addaction name="separator"/>
    <addaction name="actionRandomize"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionToggleCountdown"/>
    <addaction name="actionToggle_Player_Detail"/>
    <addaction name="separator"/>
    <addaction name="actionPlaylistNumbering"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Playlist"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About fluxplayer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRandomize">
   <property name="text">
    <string>R&amp;andomize Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAdd_tracks">
   <property name="text">
    <string>Add tracks...</string>
   </property>
  </action>
  <action name="actionDevice_configuration">
   <property name="text">
    <string>Device configuration</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionToggleCountdown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Countdown</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Repeat</string>
   </property>
  </action>
  <action name="actionOpen_playlist">
   <property name="text">
    <string>&amp;Open Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_playlist">
   <property name="text">
    <string>&amp;Save Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Playlist_As">
   <property name="text">
    <string>Save Playlist &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew_Playlist">
   <property name="text">
    <string>&amp;New Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAddMedia">
   <property name="text">
    <string>Add &amp;Media File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAddMediaFolder">
   <property name="text">
    <string>Add Media &amp;Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionAdd_Media_URL">
   <property name="text">
    <string>Add Media &amp;URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAuto_Populate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Populate</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>View &amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionToggle_Player_Detail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Player Detail</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionAudio_Settings">
   <property name="text">
    <string>A&amp;udio Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+,</string>
   </property>
  </action>
  <action name="actionPlaylistNumbering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Absolute Track Numbering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FXPlayerControlWidget</class>
   <extends>QWidget</extends>
   <header>fxplayercontrolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FXTableView</class>
   <extends>QTableView</extends>
   <header>fxtableview.h</header>
  </customwidget>
  <customwidget>
   <class>FXVerticalAudioMeterWidget</class>
   <extends>QWidget</extends>
   <header>fxverticalaudiometerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fluxplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>FluxPlayerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
